app/value_objects/employment_status.rb

```diff
@@ -7,4 +7,10 @@ class EmploymentStatus < ValueObject
     SHAREHOLDER = new(:shareholder),
     NOT_WORKING = new(:not_working),
   ].freeze
+
+  INITIAL_STATUSES = [
+    EmploymentStatus::EMPLOYED,
+    EmploymentStatus::SELF_EMPLOYED,
+    EmploymentStatus::NOT_WORKING,
+  ].freeze
```

Comment:
I have named this `INITIAL_STATUSES` indicating that the other statuses might be selected/used later on i.e. in the self-employed journey but not totally happy with it and definitely open to better suggestions for a name

Reply:
@RoseSAK What do you think of removing BUSINESS_PARTNERSHIP, DIRECTOR and SHAREHOLDER from employment_status values array and create separate  self_employed_status (app/value_objects/self_employed_status.rb) value objects ?

Reply:
Yeah that would make sense, shall we leave that for the self-employed journey though? Not sure if just removing without replacing would break any existing in progress applications if someone has selected one of the options which are removed. Although they are just redirected to use eforms anyway right?

Reply:
I don't think it will break anything. Lets go with removing BUSINESS_PARTNERSHIP, DIRECTOR and SHAREHOLDER from values array

Reply:
It probably would break in progress apps that had those selected - I've checked data and noone has one of those 3 so it's fine (right this sec)